### PR TITLE
issue: 2934322 Move CI to K8S

### PR DESCRIPTION
**Pull request by igor-ivanov**

### CODE REVIEW

File: contrib/jenkins_tests/globals.sh [hunk 1]
```diff
@@ -236,7 +203,43 @@ function do_check_result()
...
+    lnkifs=$(ip -o link | awk '{print $2,$(NF-2)}')
+    IFS=$'\n' read -rd '' -a lnk_ifs <<< "${lnkifs}"
+    IFS=${sv_ifs}
+    ifs_array=()
+
+    for nd_if in "${netdev_ifs[@]}" ; do
+        found_if=''
+        for v_if in "${lnk_ifs[@]}" ; do
+            if [ ! -z "$(echo ${v_if} | grep ${nd_if})" ] ; then
+                mac=$(echo "${v_if}"| awk '{ print $NF }') #; echo "mac=$mac"
```
Comment (ChrisCoe) [inline contrib/jenkins_tests/globals.sh:218]: Stray debug message? Another in this function as well.

File: contrib/jenkins_tests/style.sh [hunk 2]
```diff
@@ -39,18 +40,29 @@ for file in $check_files; do
...
+        i=$((i+1))
         echo "not ok $i $file # See: ${file}.diff" >> $style_tap
     else
-        echo "ok $i $file" >> $style_tap
+        rm -rf ${style_diff}
     fi
 done
+if [ $nerrors -eq 0 ]; then
+    echo "1..1" > $style_tap
+    echo "ok 1 all $(echo "$check_files" | wc -l) files" >> $style_tap
```
Comment (ChrisCoe) [inline contrib/jenkins_tests/style.sh:54]: I think you want output text to say "ok all X files" and not "ok 1 all X files"?

File: contrib/jenkins_tests/globals.sh (hunk 1 above)
Comment (igor-ivanov) [reply contrib/jenkins_tests/globals.sh:218]: what do you mean?

File: contrib/jenkins_tests/style.sh (hunk 2 above)
Comment (igor-ivanov) [reply contrib/jenkins_tests/style.sh:54]: you are right

Comment (igor-ivanov) [reply contrib/jenkins_tests/style.sh:54]: It is `tap` format. See http://testanything.org/

**Comment by igor-ivanov** [comment]
bot:retest

File: contrib/jenkins_tests/globals.sh (hunk 1 above)
Comment (ChrisCoe) [reply contrib/jenkins_tests/globals.sh:218]: I'm referring to comments `#; echo "mac=$mac"` and `#-#echo "${nd_if} --> ${if_name::-1} "`. If these were only for debugging, my suggestion is to delete them.

File: contrib/jenkins_tests/style.sh (hunk 2 above)
Comment (ChrisCoe) [reply contrib/jenkins_tests/style.sh:54]: Ah, gotcha. 

Minor suggestion: Adding a hyphen before string text would improve readability. This is done in the example in the link you shared (`ok 3 - Read the rest of the file`).

**Pull request merged**